src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlCommandTest.cs

```diff
@@ -287,37 +287,27 @@ public void CommandTimeout()
         public void CommandTimeout_Value_Negative()
         {
             SqlCommand cmd = new SqlCommand();
-            try
-            {
-                cmd.CommandTimeout = -1;
-            }
-            catch (ArgumentException ex)
-            {
-                // Invalid CommandTimeout value -1; the value must be >= 0
-                Assert.Equal(typeof(ArgumentException), ex.GetType());
-                Assert.Null(ex.InnerException);
-                Assert.NotNull(ex.Message);
-                Assert.Equal("CommandTimeout", ex.ParamName);
-            }
+
+            ArgumentException ex = Assert.Throws<ArgumentException>(() => cmd.CommandTimeout = -1);
+            // Invalid CommandTimeout value -1; the value must be >= 0
+            Assert.Equal(typeof(ArgumentException), ex.GetType());
```

Comment:
More opportunity to improve; the redundant assertion here and for the following similar ones.